layouts/partials/paige/analytics/counter.html

```diff
@@ -0,0 +1,5 @@
+{{ $page := . }}
+
+{{ with $page.Param "paige.analytics.counter" }}
+    <script src="https://cdn.counter.dev/script.js" data-id="{{ .account_id }}" data-utcoffset="-7"></script>
```

Comment:
Do we need to parameterize utcoffset?

Is this meant to be the user's default time zone offset? If so, what happens if the browser is in another time zone?

Reply:
This is a good question. I'll read the documentation more before I submit an update for this.